docs/installation.rst

```diff
@@ -1,13 +1,101 @@
+        'metagov.core',
+    ]
+
+You should also add a variable named METAGOV_SETTINGS to your ``settings.py``. This is used to configure plugins, for example:
+
+.. code-block:: python
+
+    METAGOV_SETTINGS = {
+        "SLACK": {
+            "CLIENT_ID": env("SLACK_CLIENT_ID", default=None),
+            "CLIENT_SECRET": env("SLACK_CLIENT_SECRET", default=None),
+            "SIGNING_SECRET": env("SLACK_SIGNING_SECRET", default=None),
+            "APP_ID": env("SLACK_APP_ID", default=None),
+        }
+    }
+
+Read more about :ref:`installing and configuring plugins<Plugins>`.
+
+Signal Handlers
+^^^^^^^^^^^^^^^
+
+When Metagov receives an event from a third-party platform, it emits a `platform_event_created` `Signal <https://docs.djangoproject.com/en/4.0/topics/signals/>`_. Your Driver should use signal receivers to capture these events and perform actions, for example:
+
+.. code-block:: python
+
+    from django.dispatch import receiver
+
+    @receiver(platform_event_created, sender=Github)
+    def github_event_receiver(sender, instance, event_type, data, initiator, **kwargs):
+        logger.debug(f"Received {event_type} event from {instance}")
+        # custom logic here
+
+Metagov also emits a `governance_process_updated` signal when a GovernanceProcess is updated. Again, use receivers to capture the signal, for example:
+
+.. code-block:: python
+
+    @receiver(governance_process_updated, sender=GithubIssueReactVote)
+    def github_vote_updated_receiver(sender, instance, status, outcome, errors, **kwargs):
+        ...
+
+Note that in both of these examples, we use the sender parameter in the receiver decorator to filter out events and governance process updates not relevant to a specific platform. Your Driver may not perform this kind of filtering - it is optional.
+
+.. warning::
+
+    It is easy to accidentally cause signals to be duplicated. Please do not assume all signals sent with Metagov are unique, and instead make use of `dispatch_uid <https://docs.djangoproject.com/en/4.0/topics/signals/#preventing-duplicate-signals>`_ to weed out duplicate signals.
+
+Using MetagovRequestHandler
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Metagov allows your Driver to talk to third party platforms through a class called `MetagovRequestHandler`. The handler takes care of two main types of communication: webhook events sent by the third party platform, and authentication (usually via oauth).
+
+You should not need to worry about MetagovRequestHandler too much when developing your Driver, as most of the business logic is handled by Metagov Core and Plugins. However, you will typically need to know what endpoints are generated by MetagovRequestHandler so you can provide them to the third-party platforms when setting up webhooks or oauth. The endpoints are defined in `metagov.core.urls.py`, which can be added to your project's `urls.py`. Alternatively, you can use that file as a jumping off point for customizing the endpoints.
```

Comment:
Ok this isn't how we do it in policykit fwiw. In PolicyKit we define the URLs and use the MetagovRequestHandler in the views:

https://github.com/amyxzhang/policykit/blob/d2884b6b70d6e21e894cc0f2e8774e729be148b0/policykit/policykit/urls.py#L35
https://github.com/amyxzhang/policykit/blob/d2884b6b70d6e21e894cc0f2e8774e729be148b0/policykit/policykit/urls.py#L66-L69